enforcer-rules/src/main/java/org/apache/maven/enforcer/rules/ExternalRules.java

```diff
@@ -46,11 +57,75 @@ public final class ExternalRules extends AbstractEnforcerRuleConfigProvider {
     private static final String LOCATION_PREFIX_CLASSPATH = "classpath:";
 
     /**
-     * The external rules location. If it starts with "classpath:", the resource is read from the classpath.
+     * The external rules location. If it starts with <code>classpath:</code> the resource is read from the classpath.
      * Otherwise, it is handled as a filesystem path, either absolute, or relative to <code>${project.basedir}</code>
+     *
+     * @since 3.2.0
      */
     private String location;
 
+    /**
+     * An optional location of an XSLT file used to transform the rule document available via {@link #location} before
+     * it is applied. If it starts with <code>classpath:</code> the resource is read from the classpath.
+     * Otherwise, it is handled as a filesystem path, either absolute, or relative to <code>${project.basedir}</code>
```

Comment:
unfortunately - documentation is not generated from code ...
please move it to: enforcer-rules/src/site/apt/externalRules.apt.vm